README.MD

```diff
@@ -125,6 +130,24 @@ On dev environment all queries executed by the library MongoDB\Collection class
 
 [![Profiler Panel](https://github.com/facile-it/mongodb-bundle/blob/master/docs/img/profiler_panel.png)](https://github.com/facile-it/mongodb-bundle/blob/master/docs/img/profiler_panel.png)
 
+### Driver options
+
+Example service to manage driver options. This class must implements Facile\MongoDbBundle\Services\DriverOptions\DriverOptionsInterface.
+
+```
+# config/packages/facile_it_mongodb.yaml
+mongo_db_bundle:
+    clients:
+        ...
+        driverOptions:  
+            context: { cafile: 'example/route/file.crt' }
```

Comment:
I think this is to much related to your specific problem.

It's great that you have a general way to configure driver options through a factory service. But why put extra effort to provide a shortcut for this?